about.html

```diff
@@ -46,7 +46,7 @@
       Happy Coding!
       <br>
       <br>
-      <pre> while(!(succeed = try () ));</pre>
+      <pre> while(!succeed){ try();}</pre>
```

Comment:
There's one difference though :stuck_out_tongue: - In the new edit, it assumes there is no need to try at all if you have already succeeded :laughing: (I don't think that's right though :) )